### DISCUSSION TITLE
Target SDK. 4.7.2 or newer?

### DISCUSSION THREAD

**Discussion by GronbergInfo** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3053] [Q&A]
I downloaded the code as zip, opened the demo solution in VS2022 community.
Cannot install .NET SDK 4.5 / 4.6.
Cannot get it to compile because of this, i have searched thru all files and changed to 4.7.2 where framework mentions 4.6.2

How do i get this to work in my solution if i cannot install the older SDK?

**Comment by nicolaihenriksen** [comment]
@GronbergInfo Admittedly I have not tested this, but I would assume simply going through the *.csproj files and changing the monikers in the `TargetFrameworks` element should do it.

So something like this:
```xml
<TargetFrameworks>net462;netcoreapp3.1;net6.0-windows;net7.0-windows</TargetFrameworks>
```

becomes something like this (`net462` changed to `net472`):
```xml
<TargetFrameworks>net472;netcoreapp3.1;net6.0-windows;net7.0-windows</TargetFrameworks>
```

But then again, it sounds like this is what you've already tried...
